### PR TITLE
ACC-1581 Fix payFasterNextTime reload iframe bug

### PR DESCRIPTION
**Pull request by asugar13**
### Description
The input checkbox for `payFasterNextTime` was causing the iframe to reload everytime it was clicked one. The input needs to go outside of the main div in order for this not to happen.

### Ticket
[JIRA](https://financialtimes.atlassian.net/jira/software/c/projects/ACC/boards/1461?modal=detail&selectedIssue=ACC-1581)

### Screenshots
Before:
https://user-images.githubusercontent.com/10453619/164730814-f9e6a8af-3b18-452a-9669-280e403bfb6c.mov

After:
https://user-images.githubusercontent.com/10453619/164729647-882db70a-20c2-4abe-bbce-4520f10b0421.mov

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-	extends: ['@financial-times/eslint-config-next'],
-};
+    extends: ['@financial-times/eslint-config-next'],
```
Comment (joelcarr) [inline .eslintrc.js:2]: **Question:** Is this change required to be committed?

**Review by joelcarr — COMMENTED** [review]
Looks good! Just a question about the committed whitespace change to `.eslintrc.js`.

File: .eslintrc.js (hunk 1 above)
Comment (asugar13) [reply .eslintrc.js:2]: I think I was having issues committing without this file. Doesn't look like an actual change, just linting. I can look into ommitting it from the commits though

**Review by joelcarr — APPROVED** [review]

**Review by ekocian — APPROVED** [review]

**Review by joelcarr — APPROVED** [review]
